templates/backup-pre-mysql.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+
+# Exits after first failure
+set -e
+
+TMPDIR="{{ MYSQL_SERVER_BACKUP_DIR }}"
+timestamp=$(date +%Y%m%dT%H:%M:%S)
+
+rm -rf "${TMPDIR}"
+mkdir -p "${TMPDIR}"
+mysql -s -e 'show databases' | tail -n +1 | grep -v performance_schema | grep -v information_schema | while read DBNAME; do mysqldump -uroot --complete-insert --events --single-transaction "${DBNAME}" > "${TMPDIR}/${DBNAME}".sql; done
```

Comment:
No, `tail -n +1` print everything starting from the first line, it doesn't skip anything:
```
$ seq 3
1
2
3
$ seq 3 | tail -n +1
1
2
3
```